roadmap/roadmap.md

```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
 
 ## Functionality
 
-- :large_orange_diamond: Provide next-generation spectroscopic reduction, analysis, and visualization tools usable by individual researchers and larger surveys.
+The core product of the Project is high quality Astronomical software, including both the common core package and affiliated packages. This category encompasses goals having to do with this core Project functionality.
+
+- :green_circle: Provide next-generation spectroscopic analysis tools usable by individual researchers and larger surveys.
+
+- :green_circle: Add quantity support for non-NumPy arrays.
+
+- :large_orange_diamond: Provide next-generation spectroscopic reduction tools usable by individual researchers and larger surveys.
+
+
+## Infrastructure, Documentation
+
+Underlying all Astropy packages is critial infrastucture that ensures high quality, well-documented software is released. This category emcompasses goals having to do with this infrastructure and documentation.
+
+
+- :green_circle: Implement robust performance benchmark reporting.
+
+- :large_orange_diamond: Improve documentation for infrastructure setup, as well as the developer documentation.
+
+- :large_orange_diamond: Revive dashboards for developers (CI status etc) and community (PR trends etc).
+
 
-- :large_orange_diamond: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified I/O system.
+## Interoperability and compatibility
 
-- :large_orange_diamond: Improve interoperability and compatibility, while reducing duplication, amongst Astropy-affiliated packages and other community standard software.
+One of the core principles of the Astropy Project is to "foster an ecosystem of interoperable astronomy packages." This category encompasses goals related to that principle of interoperability and compatibility.
+
+- :green_circle: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified I/O system: allowing user added classes/formats, and implementing GWCS.

```

Comment:
Clarify acronym. We don't have to capitalize "astronomy," do we?
```suggestion
- :green_circle: Improve support and validation for input/output (I/O) of standard file formats used in astronomy by improving the unified I/O system: allowing user added classes/formats, and implementing GWCS.
```

Reply:
I probably originally wrote Astropy and didn't change the capitalization when I changed the wording.